### PR TITLE
Kubeadm alpha cmd

### PR DESCRIPTION
**Pull request by fabriziopandini**
**What this PR does / why we need it**:
This PR moves under `kubeadm alpha` all the `kubeadm alpha phase` subcommands that are not part of the kubeadm init workflow.

```
kubeadm alpha certs renew *
kubeadm alpha kubeconfig user
kubeadm alpha kubelet config download
kubeadm alpha kubelet config enable-dynamic
kubeadm alpha preflight node
```

**Special notes for your reviewer:**:

´kubeadm alpha phase` will be removed when phases migration completes, 

**Which issue(s) this PR fixes** : 
Rif https://github.com/kubernetes/kubeadm/issues/1163

/sig cluster-lifecycle
/kind feature
/cc @neolit123 
/assign @timothysc 
@kubernetes/sig-cluster-lifecycle-pr-reviews 
@chuckha 
@rosti 
@ereslibre 

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by neolit123 — APPROVED** [review]
/lgtm
thanks @fabriziopandini

File: cmd/kubeadm/app/cmd/alpha/certs.go [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+	"fmt"
+
+	"github.com/spf13/cobra"
+	kubeadmscheme "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/scheme"
+	kubeadmapiv1beta1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1"
+	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/options"
+	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	certsphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/certs"
+	"k8s.io/kubernetes/cmd/kubeadm/app/phases/certs/renewal"
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/alpha/certs.go]: we need to move renewal outside of the phases namespace, IMHO.

Comment (fabriziopandini) [reply cmd/kubeadm/app/cmd/alpha/certs.go]: @neolit123 
This PR focuses on UX only, and it is already XXL, so I prefer to keep out of the field refactoring `/cmd/kubeadm/app/phases`.

Nevertheless I fully agree that we should clean up package structure, but IMO phases are a UX thing and should not be reflected outside `cmd/kubeadm/app/cmd`.
My idea is to rename `cmd/kubeadm/app/phases` into `cmd/kubeadm/app/pkg` (or in `pkg/kubeadm`, similar to other Kubernetes commnad), but this should be discussed at sig level before

File: cmd/kubeadm/app/cmd/alpha/kubeconfig.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	}
+
+	cmd.AddCommand(newCmdUserKubeConfig(out))
+	return cmd
+}
+
+// newCmdUserKubeConfig returns sub commands for kubeconfig phase
+func newCmdUserKubeConfig(out io.Writer) *cobra.Command {
+
+	cfg := &kubeadmapiv1beta1.InitConfiguration{}
```
Comment (timothysc) [inline cmd/kubeadm/app/cmd/alpha/kubeconfig.go]: I still don't like versioned-aliased references to versioned objects.  

kubeadmapi.foo should be sufficient and minimize future impact to the surrounding code when we bump versions again.

File: cmd/kubeadm/app/cmd/alpha/kubelet.go [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+		Long:  cmdutil.MacroCommandLongDescription,
+	}
+
+	cmd.AddCommand(newCmdKubeletConfigDownload())
+	cmd.AddCommand(newCmdKubeletConfigEnableDynamic())
+	return cmd
+}
+
+// newCmdKubeletConfigDownload calls cobra.Command for downloading the kubelet configuration from the kubelet-config-1.X ConfigMap in the cluster
+func newCmdKubeletConfigDownload() *cobra.Command {
```
Comment (timothysc) [inline cmd/kubeadm/app/cmd/alpha/kubelet.go]: Shouldn't this be a phase for join?

**Review by timothysc — COMMENTED** [review]
Definitely have some questions to discuss on the sig call with where we want some of the kubelet functions to live.

File: cmd/kubeadm/app/cmd/alpha/kubelet.go (hunk 3 above)
Comment (timothysc) [reply cmd/kubeadm/app/cmd/alpha/kubelet.go]: Would you open an issue and cross ref here for join phases.

**Comment by timothysc** [comment]
Per conversation on the SIG call today we agree on the path forward and should enqueue issues for 1.14. 

Please address comments then LGTM from me.

File: cmd/kubeadm/app/cmd/alpha/kubeconfig.go (hunk 2 above)
Comment (fabriziopandini) [reply cmd/kubeadm/app/cmd/alpha/kubeconfig.go]: this is actually a reference to a v1beta1 object, and flags rely on it.
Then it is converted (with the flag values assigned) to an internal object by ConfigFileAndDefaultsToInternalConfig

**Comment by fabriziopandini** [comment]
/lgtm
as per comment

**Comment by fabriziopandini** [comment]
@timothysc rebased and addressed comments, but I can't lgtm my own PR
@timothysc @neolit123 could you kindly lgtm again

**Pull request merged**